mne/io/proj.py

```diff
@@ -774,7 +806,7 @@ def make_eeg_average_ref_proj(info, activate=True, verbose=None):
         raise ValueError('Cannot create EEG average reference projector '
                          '(no EEG data found)')
     vec = np.ones((1, n_eeg))
-    vec /= n_eeg
+    vec /= np.sqrt(n_eeg)
```

Comment:
that's a bug fix?

Reply:
Sqrt is really just for consistency. All other projectors we create are unit norm, and this makes the avg ref also have unit norm (which it doesn't on main).

When we actually compute the projection matrix, we renormalize (and then later orthonormalize again, I think), which is why it doesn't usually matter in practice, and hence doesn't really need to be called a bugfix. This change won't affect the user experience or outputs at all. But when doing low level internal testing, it's helpful to have all projectors created with unit norm in the first place.